### ISSUE TITLE
pymel.ls(sl=True) on particles raises an error

### ISSUE THREAD

**Issue by elrond79** [LumaPictures/pymel#290]
_From [michael....@gmail.com](https://code.google.com/u/105768425899132608018/) on August 25, 2011 18:41:27_

Describe the problem. What Version of Maya are you using? 2011 On what operating system? (be as specific as possible and include service packs, 64bit vs 32bit, etc) 2011-1.1 x64 What Version of PyMEL are you using? 1.0.0 What is the expected output? What do you see instead? - Run ls(selection=True) on particles, get:

\# Error: 
\# Traceback (most recent call last):
\#   File "<maya console>", line 3, in \<module>
\#   File "/local/apps/linux/aw/maya2011/lib/python2.6/site-packages/pymel/core/general.py", line 896, in ls
\#     return map(PyNode, res)
\#   File "/local/apps/linux/aw/maya2011/lib/python2.6/site-packages/pymel/core/general.py", line 1543, in __new__
\#     pymelType, obj = _getPymelType( argObj, name )
\#   File "/local/apps/linux/aw/maya2011/lib/python2.6/site-packages/pymel/core/general.py", line 109, in _getPymelType
\#     pymelType = _getPymelTypeFromObject( obj, name )
\#   File "/local/apps/linux/aw/maya2011/lib/python2.6/site-packages/pymel/core/general.py", line 44, in _getPymelTypeFromObject
\#     raise MayaNodeError
\# MayaNodeError: Maya Node does not exist: u'None' # 

This is practically a dupe of Issue 95 . If possible, provide a few lines of code to reproduce the problem. It helps us if your code will reproduce the problem from a new scene. Does maya.cmds exhibit the same problem? No Please provide any additional information below.

_Original issue: http://code.google.com/p/pymel/issues/detail?id=256_

**Comment by elrond79** [comment]
_From [chr...@superbotentertainment.com](https://code.google.com/u/115022229252644175055/) on August 26, 2011 17:34:46_

it seems to work correctly for me with pymel 1.0.3, try getting the latest
ie: i can select particles and then
ls(sl=True)
ls(sl=1) also works for me
ls(ls=true) does not
c.r

**Comment by elrond79** [comment]
_From [michael....@gmail.com](https://code.google.com/u/105768425899132608018/) on August 26, 2011 17:41:25_

How can I most easily tell my version? Unfortunately I'm writing this for a tool that must be distributed at a studio and it may be a while before we can get everyone upgraded.

**Comment by elrond79** [comment]
_From [elron...@gmail.com](https://code.google.com/u/112796493857278763022/) on October 12, 2011 01:33:20_

Seems to work for me too - exact code I used to test:

import pymel.core as pm
part = pm.createNode('particle')
pm.select(part)
pm.ls(sl=1)

to tell the pymel version, you can do:

import pymel
print pymel.__version__

closing this ticket unless someone can confirm this is a problem in 1.0.3 or later

**Status:** WontFix
